lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -186,45 +291,58 @@ class CategoryRenderer {
       return element;
     }
 
-    const passedElem = this._dom.createElement('details', 'lh-passed-audits');
-    const passedSummary = this._dom.createElement('summary', 'lh-passed-audits-summary');
-    passedSummary.textContent = `View ${passedAudits.length} passed items`;
-    passedElem.appendChild(passedSummary);
-
-    for (const audit of passedAudits) {
-      passedElem.appendChild(this._renderAudit(audit));
-    }
+    const passedElements = passedAudits.map(audit => this._renderAudit(audit));
+    const passedElem = this._renderPassedAuditsSection(passedElements);
     element.appendChild(passedElem);
     return element;
   }
 
   /**
-   * @param {!Array<!ReportRenderer.AuditJSON>} audits
-   * @param {!ReportRenderer.GroupJSON} group
+   * @param {!ReportRenderer.CategoryJSON} category
+   * @param {!Object<string, !ReportRenderer.GroupJSON>} groups
    * @return {!Element}
    */
-  _renderAuditGroup(audits, group) {
-    const auditGroupElem = this._dom.createElement('details',
-          'lh-audit-group lh-expandable-details');
-    const auditGroupHeader = this._dom.createElement('div',
-          'lh-audit-group__header lh-expandable-details__header');
-    auditGroupHeader.textContent = group.title;
+  _renderPerformanceCategory(category, groups) {
+    const element = this._dom.createElement('div', 'lh-category');
+    element.id = category.id;
+    element.appendChild(this._renderCategoryScore(category));
 
-    const auditGroupDescription = this._dom.createElement('div', 'lh-audit-group__description');
-    auditGroupDescription.textContent = group.description;
+    const metricAudits = category.audits.filter(audit => audit.group === 'perf-metric');
+    const metricAuditsEl = this._renderAuditGroup(metricAudits, groups['perf-metric']);
+    metricAuditsEl.open = true;
+    element.appendChild(metricAuditsEl);
+
+    const hintAudits = category.audits
+        .filter(audit => audit.group === 'perf-hint' && audit.score < 100)
+        .sort((auditA, auditB) => auditB.result.rawValue - auditA.result.rawValue);
+    if (hintAudits.length) {
+      let maxWaste = 0;
+      hintAudits.forEach(audit => maxWaste = Math.max(maxWaste, audit.result.rawValue));
+
+      const scale = Math.ceil(maxWaste / 1000) * 1000;
+      const hintAuditsEl = this._renderAuditGroup(hintAudits, groups['perf-hint'],
+          audit => this._renderPerfHintAudit(audit, scale));
```

Comment:
should this go through our typical `render()` method?  if so, we'd need to smush `scale` into the audit object and set up the `type` etc..

that seems pretty reasonable, but lemme know if you're thinkin otherwise.
i'd say if we don't then it probably shouldn't be prefixed with `_render` as its done differently than the other `_render**()`'s